code/modules/organs/organ.dm

```diff
@@ -399,4 +414,19 @@ var/list/organ_cache = list()
 		return 0
 	if(robotic && robotic < ORGAN_LIFELIKE)	//Super fancy humanlike robotics probably have sensors, or something?
 		return 0
-	return 1
\ No newline at end of file
+	return 1
+
+/obj/item/organ/proc/handle_organ_mod_special(var/removed = 0)	// Called when created, transplanted, and removed.
+	if(!istype(owner))
+		return
+
+	if(!removed && organ_verbs)
+		for(var/verb_path in organ_verbs)
+			owner.verbs |= verb_path
+	else if(organ_verbs)
+		for(var/verb_path in organ_verbs)
+			owner.verbs -= verb_path
```

Comment:
I may be wrong, but I'm pretty sure this code means that, in the off chance you have two organs with the same verb(s), removing one will totally clear the verb(s) from your mob. Is this intended?

Reply:
Not intended, I'm just an idiot and didn't consider it.

Reply:
Should note the organs now remove their verbs if there's no other organs that add the verbs.